about.html

```diff
@@ -0,0 +1,163 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Kenya Redcross Fundraising concert</title>
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
+    <link rel="stylesheet" type="text/css" href="./css/main.css" />
+    <link rel="stylesheet" type="text/css" href="./bout.css" />
+
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.0/css/all.min.css" integrity="sha512-xh6O/CkQoPOWDdYTDqeRdPCVd1SpvCA9XXcUnZS2FmJNp1coAFzvtCN9BmamE+4aHK8yyUHUSCcJHgXloTyT2A==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <!-- Font Awesome -->
+    <link
+    href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0/css/all.min.css"
+    rel="stylesheet"
+    />
+    <!-- Google Fonts -->
+    <link
+    href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700&display=swap"
+    rel="stylesheet"
+    />
+    <!-- MDB -->
+    <link
+    href="https://cdnjs.cloudflare.com/ajax/libs/mdb-ui-kit/5.0.0/mdb.min.css"
+    rel="stylesheet"
+    />
+    
+  </head>
+  <body>
+    <div class="primary-nav">
+      <ul class="navone-items">
+        <li><a href="https://twitter.com/noqsnoq1"><i class="fa-brands fa-twitter"></i></a></li>
+        <li><a href="https://www.facebook.com/nyoqs/"><i class="fa-brands fa-facebook-f"></i></a></li>
+        <li><a href="#english">English</a></li>
+        <li><a href="#My-page">My page</a></li>
+        <li><a href="#login">Login</a></li>
+      </ul>
+    </div>
+    <main class="mainone">
+      <div class="wrapper">
+        <header class="main-body">
+          <div class="mainone1">
+            <nav class="main-nav">
+              <div class="items">
+                    <a class="logo navlogo" href="/index.html"><img src="./image/logo-krcs.png" height="60" alt="kenya redcross logo"></a>
+                    <div class="mobile" id="harmburger">
```

Comment:
Kindly modify the about-us page to match the design guideline.

According to the design, the heading only takes up 2 lines and the details area has a white background.

Your design:
![image](https://user-images.githubusercontent.com/20554583/192086539-bdadbfa0-3f92-4d3a-82fc-68ba0f853018.png)


Design Guideline:
![image](https://user-images.githubusercontent.com/20554583/192086555-2a32318f-01ae-413d-853f-83bf84162c4d.png)